client/package.json

```diff
@@ -9,6 +9,7 @@
 		"@mui/base": "^5.0.0-beta.2",
 		"@mui/icons-material": "^5.11.16",
 		"@mui/material": "^5.13.2",
+		"@reduxjs/toolkit": "^1.9.5",
```

Comment:
@marvinsjsu , do we want to settle on `redux` without any other form library? If so, we'll also want to add some cards to remove `react-query` and `react-hooks-form`. There are a few forms (signupnonprofit) using it. I like Redux' separation of concerns, but one downside is all the boilerplate.

Either way, I'm down with it.